### PR TITLE
Chore: enable bdd coverage

### PR DESCRIPTION
**Pull request by pmuir**
#### Submitter checklist

- [ ] Change is code complete and matches issue description.
- [ ] Change is covered by existing or new tests.

#### Which issue this PR fixes

fixes #

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -119,8 +121,15 @@ BUILDFLAGS += -ldflags "-X $(ROOT_PACKAGE)/pkg/features.FeatureFlagToken=$(FEATU
 RELEASE_ORG_REPO := cloudbees/cloudbees-jenkins-x-distro
 endif
 
+ifdef JUNIT_REPORT
+# REPORT_PIPE must go at end of line
+OUTFILE := $(shell mktemp)
+REPORT_PIPE := -test.v 2>&1 | tee $(OUTFILE) | go-junit-report > build/integration.junit.xml & tail -f $(OUTFILE)
```
Comment (ccojocar) [inline Makefile]: Can we also keep the HTML report?

Comment (ccojocar) [reply Makefile]: I'd like to open it in the browser. The Junit XML is hard to visualise without extra tools.

Comment (pmuir) [reply Makefile]: yeah, i assumed no one would use that, i'll add it back in

Comment (pmuir) [reply Makefile]: Actually that's the other PR I have open ;-)

**Pull request merged**